releases/release-8.5.1.md

```diff
@@ -0,0 +1,105 @@
+---
+title: TiDB 8.5.1 Release Notes
+summary: Learn about the improvements and bug fixes in TiDB 8.5.1.
+---
+
+# TiDB 8.5.1 Release Notes
+
+Release date: January xx, 2025
+
+TiDB version: 8.5.1
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v8.1/quick-start-with-tidb) | [Production deployment](https://docs.pingcap.com/tidb/v8.1/production-deployment-using-tiup)
+
+## Operating system and platform requirement changes
+
+Starting from v8.5.1, TiDB resumes testing of CentOS Linux 7 and is compatible with it. If you plan to deploy TiDB v8.5 or upgrade your cluster to v8.5, ensure you use TiDB v8.5.1 or a later version.
+
+- TiDB v8.4.0 DMR and v8.5.0 releases dropped support and testing with CentOS Linux 7 because it reaches [EOL status on June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol). Upgrading a TiDB cluster on CentOS 7 to v8.4.0 or v8.5.0 will cause the risk of cluster unavailability.
```

Comment:
```suggestion
- TiDB v8.4.0 DMR and v8.5.0 releases dropped support and testing with CentOS Linux 7 because it reached [EOL status on June 30, 2024](https://www.redhat.com/en/topics/linux/centos-linux-eol). Upgrading a TiDB cluster on CentOS 7 to v8.4.0 or v8.5.0 will cause the risk of cluster unavailability.
```